### PR TITLE
Feature/task system/81512804

### PR DESCRIPTION
**Pull request by nateng98**
- [x] Task system webapi
- [x] Db context for connecting any database
- [x] Repository for PostgreSQL
- [x] ~96% test coverage
- [ ] Update readme
- [ ] Integrate tests into pipeline

### CODE REVIEW

File: src/task_system/README.md
Comment (LeeroyDilim) [inline src/task_system/README.md:1]: include how to run tests (integration, persistence tests, yadada)

File: src/task_system/task_system_server/Models/TaskItem.cs
Comment (LeeroyDilim) [inline src/task_system/task_system_server/Models/TaskItem.cs:1]: Need enums for status and type of task

Comment (LeeroyDilim) [inline src/task_system/task_system_server/Models/TaskItem.cs:1]: Also we discussed that title meant type of task, I think its best we rename the title variable as type and use it accordingly

File: src/task_system/task_system_server/Program.cs
Comment (LeeroyDilim) [inline src/task_system/task_system_server/Program.cs:1]: check this in my pr, but I have changed some parts of this code to refer to environment variables

File: src/task_system/task_system_server/Repositories/StubTaskSystemRepository.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    private int _nextId = 1;
+
+    public StubTaskSystemRepository()
+    {
+        _tasks =
+        [
+            new TaskItem
+            {
+                Id = _nextId++,
+                Title = "Fix the bathroom leak",
```
Comment (LeeroyDilim) [inline src/task_system/task_system_server/Repositories/StubTaskSystemRepository.cs:19]: title should be switched to type of task

**Review by LeeroyDilim — COMMENTED** [review]
Good stuff, liked your advanced usage of .NET.  just a few adjustments

**Review by Mondane-Monkeys — APPROVED** [review]
Great work!

**Pull request merged**